clippy_utils/src/macros.rs

```diff
@@ -461,11 +462,13 @@ impl<'tcx> FormatArgsExpn<'tcx> {
             .collect()
     }
 
-    /// Span of all inputs
+    /// Source callsite span of all inputs
     pub fn inputs_span(&self) -> Span {
+        let format_string = self.format_string_span.source_callsite();
```

Comment:
If this is needed then I think `self.format_string_span` needs to be fixed?

Reply:
It wasn't needed there, didn't think to check individually